bin/inx-chronicle/src/api/stardust/analytics/responses.rs

```diff
@@ -50,3 +54,46 @@ pub struct StorageDepositAnalyticsResponse {
 }
 
 impl_success_response!(StorageDepositAnalyticsResponse);
+
+/// Response of `GET /api/analytics/richlist[?top=<usize>]`.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RichlistAnalyticsResponse {
+    pub distribution: Vec<DistributionStatDto>,
+    pub top: Vec<AddressStatDto>,
+}
```

Comment:
I think we should split these two queries, following the logic of the other endpoints. One should be `richest-adresses` and the other one should be `token-distribution`.

Additionally, I think we should return the `ledger_index` here too.

Reply:
Fair enough.